app/models/observations.js

```diff
@@ -50,7 +50,23 @@ export async function fetchObservations (surveyId, osmObjectId, userId) {
   };
 
   const observations = await db('observations')
-    .select()
+    .select(
+      [
+        'observations.id',
+        'observations.createdAt',
+        'observations.uploadedAt',
+        'userId',
```

Comment:
I suppose we should namespace these and adjust where ever these attributes are used on the frontend?

Reply:
Not sure how to do that with `knex` @geohacker, do you have an example?